framework/src/main/java/org/apache/drill/test/framework/TestDriver.java

```diff
@@ -252,7 +252,7 @@ public int runTests() throws Exception {
       for (DrillTestJdbc test : verificationFailures) {
         LOG.info(test.getInputFile());
         LOG.info("Query: \n" + test.getQuery());
-        LOG.info(test.getException().getMessage());
+        LOG.info(test.getException() == null? "":test.getException().getMessage());
```

Comment:
It was true for type verification. I have since changed it to also throw a verification exception. Let me revert it back so we always enforce a message.